src/mdsynthesis/limbs.py

```diff
@@ -216,7 +215,11 @@ def _apply_resnums(self):
                 resnums = None
 
         if resnums:
-            self._treant._universe.residues.set_resnums(np.array(resnums))
+            # Compatibility for MDAnalysis pre 0.16.0
```

Comment:
https://github.com/MDAnalysis/mdanalysis/blob/release-0.14.0/package/MDAnalysis/core/AtomGroup.py#L1397

I think you can just switch to only the `.resnums = resnums` branch here, it was supported in 0.14